rs/recovery/src/nns_recovery_failover_nodes.rs

```diff
@@ -380,16 +388,20 @@ impl RecoveryIterator<StepType, StepTypeIter> for NNSRecoveryFailoverNodes {
             }
 
             StepType::WaitForCUP => {
-                if let Some(node_ip) = self.params.upload_node {
+                if let Some(method) = self.params.upload_method {
+                    let node_ip = match method {
+                        UploadMethod::Remote(ip) => ip,
+                        UploadMethod::Local => IpAddr::V6(Ipv6Addr::new(0, 0, 0, 0, 0, 0, 0, 1)),
```

Comment:
Maybe we could use a named constant for this

Reply:
done